packages/cxl-lumo-styles/scss/themes/vaadin-context-menu-list-box.scss

```diff
@@ -12,3 +12,25 @@
     box-shadow: inset 0 0 0 ($vaadin-focus-ring-thickness * 0.5) var(--lumo-primary-color);
   }
 }
+
+:host([theme~="cxl-marketing-nav"]) {
+  box-shadow: none !important; /* stylelint-disable-line declaration-no-important */
+  // box-shadow: 0 0 0 1px var(--lumo-shade-5pct), var(--lumo-box-shadow-m);
+}
+
+@media screen and (min-width: 782px) {
```

Comment:
https://github.com/conversionxl/aybolit/blob/3bdb785cb663ac91dbb9e738d42f66556e5eb72e/packages/cxl-lumo-styles/scss/global.scss#L142

Same value used but as `max-width`. Should this be `783px` then?